### DISCUSSION TITLE
I am not getting eligible for Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by devarsh10** [community/community#63730] [GitHub Education]
### Hi everyone,

I am Devarsh Shah and I am 4th Year CSE Undergrad, I have sent my Id card and Academic calender to apply for Github Student Developer Pack and wanted to start my journey as Github Campus Expert. But I am getting rejection and not able to upload my id card now.
Please help me to get out of this situation.

Regards,
Devarsh Shah.

**Comment by Ntshangase** [comment]
Hey @devarsh10 

It will be easier for us to assist you if you provide more details as to why you were rejected.
